fsharp-backend/src/BackendOnlyStdLib/LibStaticAssets.fs

```diff
@@ -152,7 +152,7 @@ let fns : List<BuiltInFn> =
         (function
         | state, [] ->
           uply {
-            match! SA.latestDeployHash state.program.canvasID with
+            match! SA.getLatestDeployHash state.program.canvasID with
```

Comment:
Maybe this rename is useless? I personally find it more immediately obvious, but could also see it as too verbose. :shrug:

Reply:
Generally we have not used to get prefix, and I don't really like it.